### DISCUSSION TITLE
Can you please inplement the Static hosting services like the one from https://github.com/timlrx/tailwind-nextjs-starter-blog#deploy

### DISCUSSION THREAD

**Discussion by techhubhub** [PxlSyl/tailwind-nextjs-starter-blog-i18n#15] [General]
**Is your feature request related to a problem? Please describe.**
Feature of ssg. Thank you in advance.

**Additional context**
instructions from original  tailwind-nextjs-starter-blog

> Static hosting services / GitHub Pages / S3 / Firebase etc.
> 
> Add output: 'export' in next.config.js. See [static exports documentation](https://nextjs.org/docs/app/building-your-application/deploying/static-exports#configuration) for more information.
> 
> Comment out headers() from next.config.js.
> 
> Add unoptimized: true to the images key in next.config.js:
> 
> Alternatively, to continue using next/image, you can use an alternative image optimization provider such as Imgix, Cloudinary or Akamai. See [image optimization documentation](https://nextjs.org/docs/app/building-your-application/deploying/static-exports#image-optimization) for more details.
> 
> Remove api folder and components which call the server-side function such as the Newsletter component. Not technically required and the site will build successfully, but the APIs cannot be used as they are server-side functions.
> 
> Run yarn build. The generated static content is in the out folder.
> 
> Deploy the out folder to your hosting service of choice or run npx serve out to view the website locally.

**Comment by PxlSyl** [comment]
Hi! This is a template, and I have a lot of work these days so I can't help with this. Remember this is completely free and you must adapt to your own needs, also for things like ssg. Unless a contributor comes to help.

Thank you for your comprehension!
